samples/getting_started/QuantumHelloWorld.qs

```diff
@@ -0,0 +1,19 @@
+/// # Summary
+/// Quantum Hello World sample
+///
+/// # Description
+/// This is one of the simplest Q# programs that contains quantum part.
+/// This code prints the message then allocates a qubit and immediately measures it.
+/// Since the qubit starts in |0〉 state such measurement will always yield `Zero`.
+operation Main() : Result {
+    // Print the message. Note that some quantum computers may skip this step.
```

Comment:
I'm not sure what to make of the statement "some quantum computers may skip this step." At this point, we know that all of them will, so we should either make a stronger statement about how the `Message` function produces stdout and is ignored when compiling for quantum systems (perhaps beyond the scope of a hello world sample) or just remove this part of the comment and focus on this being an example of producing stdout from Q#.

Reply:
I'm in favor of the latter, for removal.

Reply:
I don't fully agree that we should drop it entirely. This is one of the first samples (if not the very first sample). If a person submits it to hardware, there will be no message, and it could be a surprise. I'll change this to 'when running on a simulator'. This is true (and logically doesn't preclude it from appearing when running elsewhere).